src/ts/components/seekbarcontrols.ts

```diff
@@ -50,7 +50,15 @@ const seekBarControls = (
     } else if (type === SeekBarType.Vod) {
         return arrowKeyControls(player.getCurrentTime(), { min: 0, max: player.getDuration() }, keyStepIncrements, player.seek);
     } else {
-        return arrowKeyControls(player.getVolume(), { min: 0, max: 100 }, keyStepIncrements, player.setVolume);
+        const setVolume = (volume: number) => {
```

Comment:
Why are you not using the already existing `volumeController` which is handling this?

Reply:
I need to pass the `uimanager` to the controls as well then just to use the `volumeController`. Not sure if that is really needed since the slider should only unmute the player if it is being used

Reply:
sure but why having the logic of unmuting in two places? And what I can remember there are some edge cases which are handled in the `VolumeController`. That's why we have it.